examples/AspNetCore/WebApi/MinimalOpenApiExample/Program.cs

```diff
@@ -264,18 +264,20 @@
         .Produces( 400 );
 
 app.UseSwagger();
-app.UseSwaggerUI(
-    options =>
-    {
-        var descriptions = app.DescribeApiVersions();
-
-        // build a swagger endpoint for each discovered API version
-        foreach ( var description in descriptions )
+if ( builder.Environment.IsDevelopment() )
+{
+    app.UseSwaggerUI(
+        options =>
         {
-            var url = $"/swagger/{description.GroupName}/swagger.json";
-            var name = description.GroupName.ToUpperInvariant();
-            options.SwaggerEndpoint( url, name );
-        }
-    } );
+            var descriptions = app.DescribeApiVersions();
 
+            // build a swagger endpoint for each discovered API version
+            foreach ( var description in descriptions )
+            {
+                var url = $"/swagger/{description.GroupName}/swagger.json";
+                var name = description.GroupName.ToUpperInvariant();
+                options.SwaggerEndpoint( url, name );
+            }
+        } );
+}
```

Comment:
@tdykstra diff is terrible, just added `if ( builder.Environment.IsDevelopment() )` around code.